neuro_san/service/registries_watcher/periodic_updater/registry_polling_observer.py

```diff
@@ -0,0 +1,46 @@
+# Copyright (C) 2023-2025 Cognizant Digital Business, Evolutionary AI.
+# All Rights Reserved.
+# Issued under the Academic Public License.
+#
+# You can be released from the terms, and requirements of the Academic Public
+# License by purchasing a commercial license.
+# Purchase of a commercial license is mandatory for any use of the
+# neuro-san SDK Software in commercial settings.
+#
+# END COPYRIGHT
+
+import logging
+from typing import Tuple
+from pathlib import Path
+from watchdog.observers.polling import PollingObserver
+
+from neuro_san.service.registries_watcher.periodic_updater.registry_change_handler import RegistryChangeHandler
+
+
+class RegistryPollingObserver:
+    """
```

Comment:
Polling observer.
Note that if our requested manifest update period is N seconds,
our polling cycle is computed to be shorter, around 1/4 of manifest update period,
to improve reaction to updates.